internal/commands/root.go

```diff
@@ -37,12 +36,15 @@ func NewRootCmd(use string, dockerCli command.Cli) *cobra.Command {
 				return cmd.GenBashCompletion(dockerCli.Out())
 			case "zsh":
 				return cmd.GenZshCompletion(dockerCli.Out())
-			case "":
```

Comment:
Could you add a `case len(x) > 0` to report end-user the requested shell isn't supported for completion generation ?
Just a suggestion :)

Reply:
Can't do pattern matching in Go 😛 
Add a default case with the test of `len(completion)`